cvat-core/src/api-implementation.js

```diff
@@ -199,11 +209,48 @@
             return tasks;
         };
 
-        cvat.server.installedApps.implementation = async () => {
-            const result = await serverProxy.server.installedApps();
-            return result;
+        cvat.projects.get.implementation = async (filter) => {
+            checkFilter(filter, {
+                id: isInteger,
+                page: isInteger,
+                name: isString,
+                assignee: isString,
+                owner: isString,
+                search: isString,
+                status: isEnum.bind(TaskStatus),
+            });
+
+            if ('search' in filter && Object.keys(filter).length > 1) {
+                if (!('page' in filter && Object.keys(filter).length === 2)) {
+                    throw new ArgumentError('Do not use the filter field "search" with others');
+                }
+            }
+
+            if ('id' in filter && Object.keys(filter).length > 1) {
+                if (!('page' in filter && Object.keys(filter).length === 2)) {
+                    throw new ArgumentError('Do not use the filter field "id" with others');
+                }
+            }
+
+            const searchParams = new URLSearchParams();
+            // TODO: need to check search fields
```

Comment:
What to you mean with this TODO? The fields looks to be checked with checkFilter & additional conditions above

Reply:
Forgot to check TODO was deleted